gnocchi/storage/s3.py

```diff
@@ -50,6 +50,12 @@
                  help="Maximum time to wait checking data consistency when "
                  "writing to S3. Set to 0 to disable data consistency "
                  "validation."),
+    cfg.FloatOpt('s3_max_pool_connections',
+                 min=10,
+                 default=50,
+                 help="The maximum number of connections to keep in a "
+                 "connection pool. If this value is not set, the default "
+                 "value of 10 is used."),
```

Comment:
i'm not sure the last sentence is correct. can you unset the value if there is a default?

Reply:
no, ill change that, i extracted it from the boto doc